config/config.go

```diff
@@ -0,0 +1,52 @@
+package config
+
+import (
+	"fmt"
+	"log"
+
+	"github.com/spf13/viper"
+	"gopkg.in/go-playground/validator.v9"
+)
+
+type Prometheus struct {
+	// ListenAddress to export metrics on the given port
+	ListenAddress string `mapstructure:"listen_address"`
+	// PrometheusAddress to connect to prormetheus where it has running
+	PrometheusAddress string `mapstructure:"prometheus_address"`
+}
+
+type Endpoint struct {
+	URLEndpoint string `mapstructure:"url_endpoint"`
+}
+
+type Config struct {
+	Endpoint   Endpoint   `mapstructure:"url_endpoint"`
```

Comment:
```suggestion
	RPC_Endpoint   Endpoint   `mapstructure:"url_endpoint"`
```
Let's keep it more descriptive and make this change wherever needed